docs/Developer-Guide.md

```diff
@@ -272,6 +272,9 @@ $ script -fec 'sudo -E USE_DOCKER=true ./image_builder.sh ${ROOTFS_DIR}'
 > - If you do *not* wish to build under Docker, remove the `USE_DOCKER`
 >   variable in the previous command and ensure the `qemu-img` command is
 >   available on your system.
+>   - If `qemu-img` is not installed, you will likely see errors such as `ERROR: File /dev/loop19p1 is not a block device`
+>     and `losetup: /tmp/tmp.bHz11oY851: Warning: file is smaller than 512 bytes; the loop device may be useless or invisible for system tools.`
+>     These can be mitigated by installing `qemu-img`. For example, on Debian-based platforms, use the command `apt-get update && apt-get install -y qemu-utils`
```

Comment:
Nit: I'd tend not to specify the commands to run since they might change (`apt` is a good alternative for example), but also you then need to document that the command should be run as `root`. All our docs should imply this by prefixing with `sudo` but to cover a few more bases, you could reword slightly to something like:

```suggestion
>     These can be mitigated by installing the `qemu-img` command (available in the `qemu-img` package on Fedora or the `qemu-utils` package on Debian and Ubuntu.
```

Reply:
OK, that's fair - and shows why documentation is ever-evolving and that collaboration is good - thanks for the feedback, will update shortly 🙇

Reply:
@jodh-intel as per the above, I've further amended the change; however, that's yet another commit to be squashed, and I did make the doofus error of getting my commit titles wrong in two cases; does it make more sense for me to nuke this PR and its commits, and do it again but properly this time 🤣  ?

Reply:
@jodh-intel quick question, do you want me to re-do this PR, with one rather than oodles of commits ? If so, I'll nuke it and do it again, but properly this time 🤣